openreview/venue/group.py

```diff
@@ -144,6 +144,8 @@ def create_venue_group(self):
             'public_submissions': { 'value': self.venue.submission_stage.public },
             'public_withdrawn_submissions': { 'value': self.venue.submission_stage.withdrawn_submission_public },
             'public_desk_rejected_submissions': { 'value': self.venue.submission_stage.desk_rejected_submission_public },
+            'submission_email_template': { 'value': '' },
+            'email_pcs_on_submission': { 'value': self.venue.submission_stage.email_pcs },
```

Comment:
rename the field name to `submission_email_pcs` so it follows the same format as `review_emails_pcs`, `comment_emails_pcs`, etc